src/main/java/com/google/collegeplanner/data/Meeting.java

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.package com.google.collegeplanner.data;
+
+package com.google.collegeplanner.data;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class Meeting {
+  private ArrayList<String> days;
+  private String room;
+  private String building;
+  private int startTime;
+  private int endTime;
+
+  public Meeting(String days, String room, String building, int startTime, int endTime) {
+    this.days = new ArrayList<String>();
+    this.room = room;
+    this.building = building;
+    this.startTime = startTime;
+    this.endTime = endTime;
+
+    if (days.toUpperCase().contains("M")) {
+      this.days.add("Monday");
+    }
+    if (days.toUpperCase().contains("TU")) {
+      this.days.add("Tuesday");
+    }
+    if (days.toUpperCase().contains("W")) {
+      this.days.add("Wednesday");
+    }
+    if (days.toUpperCase().contains("TH")) {
+      this.days.add("Thursday");
+    }
+    if (days.toUpperCase().contains("F")) {
+      this.days.add("Friday");
+    }
+  }
+
+  public boolean conflicts(Meeting other) {
+    // Checks to see if the two meetings occur on the same day
+    // If not, no need to check the time as they cannot conflict
+    boolean canSkip = true;
+    for (String day : other.getDays()) {
+      if (days.contains(day)) {
+        canSkip = false;
+        break;
+      }
+    }
+
+    if (canSkip) {
+      return false;
+    }
+
+    if (startTime > other.getStartTime()) {
+      if (startTime > other.getEndTime()) {
```

Comment:
should this be `>=`? These edge conditions are tricky, and we should have tests that cover every combination of this. I didn't exhaustively go through the rest here, but please double check whether > or >= is the right thing to use for each condition in this function.

I think if you have a 3-4pm (other), and 4-5pm, the first check will pass (4 > 3 == true), but the second will not (4 > 4 == false), which would return `true` by default. This doesn't actually conflict right?